### ISSUE TITLE
💥 'install' should install 'stable' instead of 'nightly' for operators/demos/etc...

### ISSUE THREAD

**Issue by fhennig** [stackabletech/stackablectl#212]
### Acceptance criterea

- [ ] Documentation updated where the behaviour changes
- [ ] `operator install` installs _stable_ by default
- [x] `demo install` install _stable_ by default

See also: https://github.com/stackabletech/stackable-cockpit/pull/389

**Comment by NickLarsenNZ** [comment]
As should `demo` install.

**Comment by sbernauer** [comment]
Just to add: `demo install` installs the version the demo is build against. This could be every possible release (including `dev`).
That's what [this](https://github.com/stackabletech/demos/blob/1a6a46dbeb1644d189b356486a5d0b3cad1d7809/stacks/stacks-v2.yaml#L4) field is for

**Comment by sbernauer** [comment]
Maybe unpopular opinion: I prefer to leave "operator install" at the nightly

**Comment by NickLarsenNZ** [comment]
> Maybe unpopular opinion: I prefer to leave "operator install" at the nightly

I disagree. It should do _stable_ stuff by default. Developers/users needing _nightly_ should be opting in.
